modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -132,11 +134,25 @@ Due to internal database restrictions, you cannot use the following keywords to
 
 . From Anypoint Monitoring:
 .. Click *Custom Metrics*.
-.. Click *Custom Metrics Connector* to open the connector on Anypoint Exchange.
+.. Click *Custom Metrics Connector* for the latest Mule version to open the connector on Anypoint Exchange.
 .. Click *Dependency Snippet*, and copy the Maven snippet.
-. In Anypoint Studio, add the snippet for the connector to your project's `pom.xml` file. 
+. In Anypoint Studio, add the `<dependency/>` snippet for the latest version of the connector to your Mule project's `pom.xml` file within the `<dependencies/>` element, for example:
++
+----
+<dependencies>
+...
+  <dependency>
+    <groupId>com.mule.modules</groupId>
+    <artifactId>mule-custom-metrics-extension</artifactId>
+    <version>2.3.1</version>
+    <classifier>mule-plugin</classifier>
+  </dependency>
+</dependencies>
+-----
++
+Your connector version value might differ from the example. After you add the dependency, the connector becomes available in the Studio palette. 
 +
-The connector is now available in the Studio palette.
+If you encounter issues with this step, see xref:studio::add-modules-in-studio-to.adoc[] to add the connector from Exchange through the Studio pallet.
```

Comment:
```suggestion
If you encounter issues with this step, see xref:studio::add-modules-in-studio-to.adoc[] to add the connector in Studio from Exchange.
```

Reply:
I'm just not sure "Studio pallet" is right. Maybe it is! You can ignore if it's fine.